src/languages/es.ts

```diff
@@ -2217,6 +2217,100 @@ export default {
             noAccountsFound: 'No se ha encontrado ninguna cuenta',
             noAccountsFoundDescription: 'Añade la cuenta en Xero y sincroniza de nuevo la conexión.',
         },
+        netsuite: {
+            export: 'Export',
```

Comment:
Let's remove this and use the generic accounting descriptions instead please. https://github.com/Expensify/App/blob/9ee8806a4011174bd215e0f950a547bf2a2c5243/src/languages/es.ts#L2411-L2423

Reply:
In fact, any key or copy that is shared with Xero or QBO above, let's move them both out of here and into the generic `accounting`